internal/app/mistapiconnector/mistapiconnector_app.go

```diff
@@ -473,14 +484,41 @@ func (mc *mac) generateRouteKeys(stream *livepeer.CreateStreamResp) []string {
 	return keys
 }
 
+func (mc *mac) sign(sharedSecret string) string {
+	s := sha256.New()
+	s.Write([]byte(sharedSecret))
+	signature := hex.EncodeToString(s.Sum(nil))
+	return signature
```

Comment:
Our signature is not just a SHA258, it's an HMAC of the message with the encryption key (hash of the encrypted payload?). This implementation must be compatible with our regular webhooks implementation, so we should change this to match it.

https://github.com/livepeer/livepeer-com/blob/6b86fb011c94e9c6dee5c9837110e0bdf7f08663/packages/api/src/controllers/helpers.ts#L19